projectorganizer/src/prjorg-main.c

```diff
@@ -115,10 +115,10 @@ static void on_project_dialog_confirmed(G_GNUC_UNUSED GObject * obj, GtkWidget *
 static void on_project_dialog_close(G_GNUC_UNUSED GObject * obj, GtkWidget * notebook,
 		G_GNUC_UNUSED gpointer user_data)
 {
-	if (page_index != -1)
+	if (properties_tab)
 	{
-		gtk_notebook_remove_page(GTK_NOTEBOOK(notebook), page_index);
-		page_index = -1;
+		gtk_container_remove(GTK_CONTAINER(notebook), properties_tab);
```

Comment:
One is too lazy to read the function description when gtk_container_remove() seems so self-descriptive :-). Will be fixed.